app/helpers/user_helper.rb

```diff
@@ -0,0 +1,40 @@
+module UserHelper
+  COLLEGE_TYPES = [
+    'College/University (4)',
+    'Technical/Community College (2)',
+    'Career School/For-Profit (2)'
+  ].freeze
+  HIGH_SCHOOL_TYPES = ['High School'].freeze
```

Comment:
what is the purpose of these single-item arrays?

Can the below case statements just be:

```
    when *COLLEGE_TYPES
      :college
    when 'High School'
      :high_school
    when 'K-12 School'
      :k12_school
    when 'Home School'
      :home_school
```

or do you expect these types might grow one day?

what I'm suggesting might be really un-ruby like, not sure :)